docs/configuration.rst

```diff
@@ -654,12 +660,20 @@ Up to infrasim-compute commit `a02417c3 <https://github.com/InfraSIM/infrasim-co
 
 .. _yamlComputeStoragebackendControllerDrivesSize:
 
+- **compute:storage_backend:-:controller:drives:-:page-file**
+
+    This option allows user to specify drive page data, which can provide addtional information for client OS.
```

Comment:
page data sounds a little bit general. how about a little bit more elaboration about this is supporting defining/injecting mode sense and inquiry data in any pages.